compiler/codegen/library.cpp

```diff
@@ -306,6 +347,46 @@ void codegen_library_python(std::vector<FnSymbol*> functions) {
   }
 }
 
+void codegen_library_fortran(std::vector<FnSymbol*> functions) {
+  if (fLibraryCompile && fLibraryFortran) {
+    setupFortranTypeMap();
+    makeFortranModule(functions);
+  }
+}
+
+void makeFortranModule(std::vector<FnSymbol*> functions) {
+  const char* filename = fortranModulename[0] != '\0' ? fortranModulename
+                                                      : libmodeHeadername;
+  int indent = 0;
+  fileinfo fort = { NULL, NULL, NULL };
+
+  openLibraryHelperFile(&fort, filename, "f90");
```

Comment:
Is there a reason for using the `.f90` extension?  Fortran 90 is old and does not include the C interface.  The version of gfortran on the user's system might not limit the language version based on the filename extension, but for documentation's sake, it might be better to use `.f` if that doesn't conflict with something else we do.

Reply:
> Is there a reason for using the `.f90` extension? Fortran 90 is old and does not include the C interface. The version of gfortran on the user's system might not limit the language version based on the filename extension, but for documentation's sake, it might be better to use `.f` if that doesn't conflict with something else we do.

My understanding was that `.f` typically means fixed-form Fortran, while `.f90` typically means free-form Fortran, even for newer standards.  My understanding was based on pages like: [fortranwiki](http://fortranwiki.org/fortran/show/File+extensions), [degenerateconic](http://degenerateconic.com/fortran-file-extensions/), [intel](https://software.intel.com/en-us/blogs/2013/01/11/doctor-fortran-in-source-form-just-wants-to-be-free).

I tried compiling a free-form program with a `.f` extension with PrgEnv-[cray|gnu|intel|pgi], and I get syntax errors for all of them because they all expect `.f` to mean fixed-form. They all compile it without a problem when the extension is `.f90`. I understand that this is not part of the standard, but it seems to be a nearly universal convention.

Reply:
OK, that's enough of a reason to convince me.